about/index.html

```diff
@@ -194,8 +143,8 @@ <h4>Bio-Formats</h4>
             </div>
             <div class="medium-4 columns">
                 <h3><img src="{{ site.baseurl }}/img/logos/ome-files-logomark.svg" alt="OME Files logo" class="logomark" /></h3>
-                <h4>OME Files</h4>
-                <p>A reference implementation that reads and writes OME-TIFF, an open file format that combines the popular, broadly supported TIFF format for storing binary pixel data, with an OME-based XML metadata header.</p>
+                <h4>OME-NGFF</h4>
```

Comment:
This equates the OME-Files logo with NGFF. I'd be careful to not do that until we're ready. Can we leave the header but then have OME-Zarr and OME-TIFF as subpoints?

Reply:
Done in [a0d6e19](https://github.com/ome/www.openmicroscopy.org/pull/784/commits/a0d6e19b304fd04b797607ce9003ec279cad42a6)